src/__/load.php

```diff
@@ -77,6 +77,7 @@
  * @method static bool isNumber(mixed $value) <p>Check if give value is number or not.</p>
  * @method static bool isObject(mixed $value) <p>Check if give value is object or not.</p>
  * @method static bool isString(mixed $value) <p>Check if give value is string or not.</p>
+ * @method static bool isEqual(mixed $value, mixed $value) <p>Check if the two values are deeply equal or not.</p>
```

Comment:
Might need to run the doc script again, the doc string for this doesn't match the updated phpdocs

Reply:
Thx, I'll go RTFM